go.mod

```diff
@@ -5,49 +5,50 @@ go 1.21
 toolchain go1.21.0
 
 require (
-	github.com/BurntSushi/toml v1.3.2
+	github.com/BurntSushi/toml v1.4.0
 	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/coreos/go-semver v0.3.1
 	github.com/go-sql-driver/mysql v1.7.1
 	github.com/google/uuid v1.6.0
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pingcap/check v0.0.0-20211026125417-57bd13f7b5f0
-	github.com/pingcap/errors v0.11.5-0.20240318064555-6bd07397691f
-	github.com/pingcap/failpoint v0.0.0-20220801062533-2eaa32854a6c
-	github.com/pingcap/kvproto v0.0.0-20240227073058-929ab83f9754
+	github.com/pingcap/errors v0.11.5-0.20240620035150-b3526907a872
+	github.com/pingcap/failpoint v0.0.0-20240527053858-9b3b6e34194a
+	github.com/pingcap/kvproto v0.0.0-20240716095229-5f7ffec83ea7
 	github.com/pingcap/log v1.1.1-0.20240314023424-862ccc32f18d
-	github.com/pingcap/tidb v1.1.0-beta.0.20240428083427-66ba419636ce
-	github.com/pingcap/tidb/pkg/parser v0.0.0-20240428083427-66ba419636ce
+	github.com/pingcap/tidb v1.1.0-beta.0.20240819080911-f751f7307761
+	github.com/pingcap/tidb/pkg/parser v0.0.0-20240819080911-f751f7307761
 	github.com/pingcap/tiflow v0.0.0-20240511022209-2bc8f02105f9
 	github.com/siddontang/go v0.0.0-20180604090527-bdc77568d726
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.9.0
-	github.com/tikv/pd/client v0.0.0-20240322051414-fb9e2d561b6e
+	github.com/tikv/pd/client v0.0.0-20240805092608-838ee7983b78
 	go.etcd.io/etcd/client/v3 v3.5.12
 	go.etcd.io/etcd/tests/v3 v3.5.12
 	go.uber.org/atomic v1.11.0
 	go.uber.org/zap v1.27.0
-	google.golang.org/grpc v1.62.1
```

Comment:
should we upgrade it in another PR, also for `github.com/BurntSushi/toml`.

Reply:
So for grpc and toml you want separate PR's?

Reply:
Yes, I recommend keeping it single-purpose

> So for grpc and toml you want separate PR's?